tests/python/pytorch/graphbolt/test_feature_store.py

```diff
@@ -1,13 +1,28 @@
+import numpy as np
 import pytest
 import torch
 from dgl import graphbolt as gb
 
 
-def test_in_memory_feature_store():
+def to_on_disk_tensor(name, t):
+    path = f"/tmp/{name}.npy"
+    t = t.numpy()
+    np.save(path, t)
```

Comment:
save as numpy instead of `torch.save()` looks a bit not intuitive here. the reason using numpy here is only numpy supports reference to disk tensor?

Reply:
Yes. I cannot find a interface of Pytorch that supports on-disk memory. And it is a common practice to use numpy memory mapped array as the backend of Pytorch on-disk tensor.

Reply:
then could we add some comments here to help people understand it?

Reply:
Added a comment. This comment is also necessary when I write `load_feature_store`.